admin/modules/banners/includes/class-template.php

```
@@ -562,6 +562,61 @@ private function get_storage_key() {
+					'law'            => isset( $settings['applicableLaw'] ) ? $settings['applicableLaw'] : 'gdpr',
+					'do_not_sell'    => $do_not_sell,
+					'optout_popup'   => ! empty( $config['optoutPopup']['status'] ),
+					// Toggling per-service / per-cookie consent changes the
+					// preference-center structure, so it must invalidate the cache.
+					'per_service'    => ! empty( $banner_control['per_service_consent'] ),
+					'per_cookie'     => ! empty( $banner_control['per_cookie_consent'] ),
+					'description'    => md5( $notice_description ),
+				)
+			)
+		);
+	}
```

PHPMD (2.15.0)

[warning] Avoid unused private methods such as 'get_layout_signature'. (undefined)

(UnusedPrivateMethod)